getting-started/gcp/main.tf

```diff
@@ -1,55 +1,72 @@
 provider "google" {
-  project = "terraform-project"
-  region  = "us-central1"
-  zone    = "us-central1-c"
+  credentials = file("<NAME>.json")
```

Comment:
If we're reading the project from a variable, can the credentials filename also be read from a variable? As-is, this does not pass `terraform validate`.

Reply:
Good idea, will refactor as suggested.